powerfulseal/execute/remote_executor.py

```diff
@@ -24,17 +24,18 @@ class RemoteExecutor(object):
         Assumes password-less setup.
     """
 
-    PREFIX = ["sh", "-c"]
+    PREFIX = ["bash", "-c"]
```

Comment:
While `bash` is generally available on most systems, I don't think you can assume it will be there on the destination node. 

`sh` is guaranteed to be on all systems.